### ISSUE TITLE
Tag 2.1.3 fails to build on Windows

### ISSUE THREAD

**Issue by j-o** [cginternals/glbinding#234]
Due to a bug in the CMake script, the gcc template API header containing `__attribute__((visibility("default")))` is selected on Windows.

The bug is fixed on master, but maybe you should think about retracting that release or at least add this to the known issues.

**Comment by scheibel** [comment]
Valid objection; I'll fix this.

**Comment by scheibel** [comment]
I probably fixed this in the v2.1 branch. Can you verify the issue is fixed before I release 2.1.4?

**Comment by j-o** [comment]
Now I get this error during cmake configure:
```
CMake Error: File D:/projects/glbinding/source/codegeneration/template_msvc_api.h.in does not exist.
CMake Error at cmake/GenerateTemplateExportHeader.cmake:8 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  source/glbinding/CMakeLists.txt:176 (generate_template_export_header)
```

Commit 567f9de on master fixes that.
Shouldn't `META_CMAKE_INIT_SHA` be updated?

**Comment by amc522** [comment]
I too was running into the issue of failing to build 2.1.3 with msvc. I tested master and this is all seems fixed. Would it be possible to make a 2.1.4 tag so there is a stable release for windows? It would be nice to upgrade the version in vcpkg.

**Comment by scheibel** [comment]
I just released glbinding-2.1.4 (https://github.com/cginternals/glbinding/releases/tag/v2.1.4). I hope everything is fixed.

**Comment by amc522** [comment]
Thanks! Just downloaded and built 2.1.4. Everything looks fine on windows.
